src/index.ts

```diff
@@ -400,22 +595,30 @@ export default class OneSignal {
         RNOneSignal.setInAppMessageLifecycleHandler();
     }
 
-    // Pass a String key and any value and creates a trigger map to pass to addTriggers()
-    static addTrigger(key, value) {
+    /**
+     * Add an In-App Message Trigger.
+     * @param  {string} key
+     * @param  {string} value
+     * @returns void
+     */
+    static addTrigger(key: string, value: string): void {
         if (!isObjectNonNull(RNOneSignal)) return;
 
         if (!key || !value) {
             console.error("OneSignal: addTrigger: must include a key and a value");
         }
 
-        let trigger = {};
+        let trigger: { [key: string]: string } = {};
```

Comment:
value can be number or bool as well https://documentation.onesignal.com/docs/iam-sdk-methods#addtrigger-method

Reply:
Done